docs/maya/simple_setup.md

```diff
@@ -142,13 +144,18 @@ To change the fiber size or its color, go to the Attribute Editor, in the debug
   <figcaption><b>Figure 15</b>: Muscle fibers combing.</figcaption>
 </figure>
 
-Finally, paint Slide On Segment Constraints (if added). It is recommended to paint only the vertices that are not attached to the rig. In this example, the tendons are painted with a value of 0.0, while the rest of the shape is painted to 1.0.
+Finally, paint Slide On Segment or Slide On Geometry Constraints (if added). It is recommended to paint only the vertices that are not attached to the rig. In this example, the tendons are painted with a value of 0.0, while the rest of the shape is painted to 1.0.
```

Comment:
I would say something like "to 1.0 or lower values", check that the screenshot for the slide on geo does not only use values of 1.0.

Reply:
Right, added.